backend/app/src/chat/chat.service.ts

```diff
@@ -202,4 +215,29 @@ export class ChatService {
   async findBlockedMeUsers(uid: number): Promise<number[]> {
     return await this.userService.findBlockedByUid(uid)
   }
+
+  async changeRoomPass(roomId: number, password: string, newPassword: string) {
```

Comment:
```suggestion
  async changeRoomPassword(roomId: number, password: string, newPassword: string) {
```
password 변경에 대한 함수라는것을 좀 더 쉽게 알아볼 수 있게 바꾸면 좋을것같아요